RELEASENOTES.docu

```diff
@@ -1748,4 +1748,12 @@ extern typedef struct { } my_type_t;</pre> </add-note></build-id>
       are themselves constant inlined method parameters (i.e. an inline method
       call propagates an inline parameter to an inline method call of its own.)
   </add-note></build-id>
+  <build-id _6="next" _7="next"><add-note> The behaviour of suppressing the
+      warning <tt>WLOGMIXUP</tt> by default below Simics API version 7 is now
+      controlled by the compatibility feature <tt>suppress_WLOGMIXUP</tt>
+      rather than being built into DMLC. This is in order to increase visibility
```

Comment:
Two notes, neither blocks a merge:

* "rather than": The compat feature _is_ still built into DMLC
* The text can be significantly shortened, since the change itself is not so significant to an end-user. I suppose it would suffice with smth like "added a compat feature that controls the default setting of WLOGMIXUP. This was mainly done to raise the visibility of its API-dependent behaviour".

Reply:
I don't really agree with point (2). In particular, I want to keep the sentence "This is in order to increase visibility of that API-dependent behaviour, and to provide migration documentation about it" completely. This will draw the attention of those who are undergoing migration efforts.
The "as a consequence" part could be stripped away, I suppose; I added it to provide more clarity about the behavior. In particular, I wanted to reassure people that were already aware of `WLOGMIXUP` that "don't worry, `--warn=WLOGMIXUP` will still work as before, but now `--no-compat` is also an alternative."

Point 1: I could strip everything past "rather than", but I want to be clear that the mentioned behavior is not new. Maybe just rewrite it to be more accurate: "... rather than being implemented ad-hoc"?

Reply:
Actually point 1 can be addressed by tweaking the final statement of the releasenote.

Reply:
Done. Is the newer one better?

Reply:
I will drop this debate. If you can find formulations that *you* think are improvements based on by comments then please do, but the current note is also good enough.

Reply:
well aright